lib/compiler.js

```diff
@@ -11,7 +11,7 @@
 const chalk = require('chalk');
 const webpack = require('webpack');
 
-const run = ({ config, watchConfig }, log) => {
+const run = ({ config, watchConfig }, log, emitJson) => {
```

Comment:
```suggestion
const run = ({ config, emitJson, watchConfig }, log) => {
```

because this is an option (and not an object like `log`) let's add this to the options argument.